public/toolkit/styles/molecules/_react-tables.scss

```diff
@@ -441,7 +441,6 @@ $narrow-column-width: 120;
   .tr-fixed-right,
```

Comment:
I think this is only used for the lease up applications table. Should we remove it here if you're nesting it under rt-table-status?